### PR TITLE
Use Algolia search indexes based on Vercel environments

### PR DESCRIPTION
**Pull request by unamashana**
## Change description

Pick search indexes based on the Vercel env

## Type of change

- [ ] Bug (fixes an issue)
- [ ] Enhancement (adds functionality)

## Related issues

> Fix [#1]()

### Development

- [ ] Lint rules pass locally
- [ ] Application changes have been tested thoroughly
- [ ] Automated tests covering modified code pass

### Code review

- [ ] Pull request has a descriptive title and context useful to a reviewer. Screenshots or screencasts are attached as necessary
- [ ] "Ready for review" label attached and reviewers assigned
- [ ] Changes have been reviewed by at least one other contributor
- [ ] Pull request linked to task tracker where applicable

### CODE REVIEW

**Comment by unamashana** [comment]
The preview deployment updates the preview index. Ideally this should be created/destroyed for each preview branch, but this will do for now!

File: docs/vercel-ignore-step.sh [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+else
+  git remote add origin "https://github.com/$VERCEL_GIT_REPO_OWNER/$VERCEL_GIT_REPO_SLUG.git"
+  git fetch origin main
+  git diff --quiet main HEAD -- ../docs/ ../openapi/
+  CHANGED=$?
+  echo "git diff --quiet main HEAD -- ../docs/ ../openapi/: $CHANGED"
+
+  if [[ $CHANGED == 0 ]]; then
+    echo "🛑 Build cancelled"
```
Comment (unamashana) [inline docs/vercel-ignore-step.sh:17]: @3v0k4 I'd add why the build was canceled.

Comment (3v0k4) [reply docs/vercel-ignore-step.sh:17]: 👍 https://github.com/magicbell-io/public/pull/48

**Comment by 3v0k4** [comment]
Thanks for fixing this 🙏 It was on my list.

**Pull request merged**